imports/ui/Editor.jsx

```diff
@@ -103,7 +101,7 @@ export class Editor extends Component {
     this.editor.on('fileUploadRequest', function (evt) {
       let upload = evt.data.requestData.upload
       let file = upload.file
-      let reader = new FileReader()
+      let reader = new window.FileReader()
```

Comment:
Not familiar enough with this to know if it's the right thing to do. What is the idea of using window.f() ?

Reply:
FileReader, Image, etc. are all global HTML5/browser objects that only exist within the browser. If you want access to these global variables, you can either explicitly use `window.globalObject`or simply have the browser automatically resolve `globalObject`.

The issue that non-browser javascript has no concept of a "window", so these HTML5-specced objects do not exist. You fairly could argue that because this file is served client side, the linter should try to be "smart" and try to resolve the dependency (some do), but Standard.js says to "Always prefix browser globals with window."

https://standardjs.com/rules.html